en/docs/references/ballerina-expression-syntax.md

```diff
@@ -380,7 +379,7 @@ To create a nested JSON object, use the following:
 
 JSON array values must be of any of the json types: `string`, `number`, `boolean`, `object`, `array`, or `null`. You can define a JSON array in one of the following two ways:
     - Using a [`json` type variable](#simple-values-in-JSON).  
```

Comment:
Bullet points are not rendering properly it seems

Reply:
Fixed.